fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -461,6 +461,10 @@ public void scenario18Test1() throws IOException, UnsupportedEncodingException {
         logger.debug("user18 can patch - SPARQL INSERTs (ACL append): {}", id);
         assertEquals(HttpStatus.SC_NO_CONTENT, getStatus(requestPatch));
 
+        requestPatch.setHeader("Content-type", "application/sparql-update; charset=UTF-8");
```

Comment:
Please add a comment stating why you added the charset here so it doesn't disappear in the future